public/js/shopping-list-item.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+class ShoppingListItem {
+  constructor(name, description) {
+    this.name = name;
+    this.description = description;
+    this.isDone = false;
+  }
+
+  check() {
+    return (this.isDone = true);
+  }
+
+  uncheck() {
+    return (this.isDone = false);
+  }
+
+  render(idx) {
+    if (this.isDone === true) {
+      return `<li class="completed_${
+        this.isDone
+      }"><button onclick="removeButtonClicked(${idx})">x</button><input type="checkbox" onchange="changeCheckedStatus(${idx}, this)" checked/><span>${
```

Comment:
Same comment as the other file with the same name as this file. This could be done with one string template literal.